pkg/registry/apis/alerting/notifications/receiver/conversions.go

```diff
@@ -79,12 +85,16 @@ func convertToDomainModel(receiver *model.Receiver) (definitions.GettableApiRece
 	}
 
 	for _, integration := range receiver.Spec.Integrations {
+		data, err := integration.Settings.MarshalJSON()
+		if err != nil {
+			return definitions.GettableApiReceiver{}, fmt.Errorf("integration '%s' of reciever '%s' is invalid: failed to convert unstructured data to bytes: %w", integration.Type, receiver.Name, err)
+		}
 		grafanaIntegration := definitions.GettableGrafanaReceiver{
 			Name:         receiver.Spec.Title,
 			Type:         integration.Type,
-			Settings:     definitions.RawMessage(integration.Settings),
+			Settings:     definitions.RawMessage(data),
 			SecureFields: integration.SecureFields,
-			//Provenance:   "", //TODO: Convert provenance?
```

Comment:
this API is non-provisioing. Therefore, provenance status should always be empty